### PR TITLE
Make Service-Workers reference informative

### LINKED ISSUE #2843: SERVICE-WORKERS reference
Is this really normative?

### PR DESCRIPTION
**Pull request by sbingler**
Closes #2843

### CODE REVIEW

**Comment by sbingler** [comment]
@mikewest Ping

File: draft-ietf-httpbis-rfc6265bis.md [hunk 1]
```diff
@@ -220,6 +207,19 @@ informative:
   HttpFieldNameRegistry:
     title: "Hypertext Transfer Protocol (HTTP) Field Name Registry"
     target: https://www.iana.org/assignments/http-fields/
+  SERVICE-WORKERS:
+    target: http://www.w3.org/TR/service-workers/
```
Comment (mikewest) [inline draft-ietf-httpbis-rfc6265bis.md:211]:
```suggestion
    target: https://www.w3.org/TR/service-workers/
```

File: draft-ietf-httpbis-rfc6265bis.md [hunk 2]
```diff
@@ -220,6 +207,19 @@ informative:
...
+    author:
+    -
+      ins: A. Russell
+      name: Alex Russell
+    -
+      ins: J. Song
+      name: Jungkee Song
+    -
+      ins: J. Archibald
+      name: Jake Archibald
```
Comment (mikewest) [inline draft-ietf-httpbis-rfc6265bis.md:222]:
```suggestion
    -
      ins: J. Archibald
      name: Jake Archibald
    -
      ins: M. Kruisselbrink
      name: Marijn Kruisselbrink
```

**Review by mikewest — APPROVED** [review]
LGTM % nits.

**Pull request merged**